packages/app/src/app/store/providers/ModelManager/ModelManager.ts

```diff
@@ -0,0 +1,24 @@
+interface IModel {
```

Comment:
This was my initial idea, but putting something like this in an effect would be ideal.